motoman_msgs/srv/WriteGroupIO.srv

```diff
@@ -0,0 +1,21 @@
+
+# Write 'value' to the Group IO element at 'address'.
+#
+# This service does not return anything.
+#
+# Addresses are plain, base-10 integers, as used and displayed by the controller
+# (on the teach pendant for instance).
+#
+# Only the following addresses can be written to:
+#
+#  - 27010 and up : Network Inputs (25010 and up on DX100 and FS100)
+#  - 10010 and up : Universal/General Outputs
```

Comment:
These are of course not the correct addresses, but I don't have the documentation with me right now.

What are the Group IO addresses the MotoROS service supports? @ted-miller @EricMarcil ?

`SimpleMessage.h` has this:

https://github.com/ros-industrial/motoman/blob/01f3e6d95a56cccf5d9f248da8e8999c48604323/motoman_driver/MotoPlus/SimpleMessage.h#L334

Reply:
DX100 and FS100:
2501 and up
1001 and up

Other controllers:
2701 and up
1001 and up

As for the upper limit, it varies based on the controller model.
https://github.com/ros-industrial/motoman/blob/01f3e6d95a56cccf5d9f248da8e8999c48604323/motoman_driver/MotoPlus/IoServer.h#L61-L268

Reply:
Ah, right. It's the grouped version of the regular I/O.

> As for the upper limit, it varies based on the controller model.

we can just refer to the controller documentation for that. This ROS service is no different from using the IOs with any other (external) interface.